draft-vandijk-dprive-ds-dot-signal-and-pin/yardsticks/draft-ietf-dprive-phase2-requirements-01.md

```diff
@@ -0,0 +1,117 @@
+This document quotes draft-ieft-dprive-phase2-requirements-01 to mark each 'requirement' as satisfied by the draft, unsatisfied by the draft, or commented otherwise. Every 'Yes' should be taken as 'when deployed by the relevant operators', or in other words 'the draft protocol permits it and the operator can choose to deploy'.
+
+> DNS Privacy Requirements for Exchanges between Recursive Resolvers and Authoritative Servers
+>
+>     draft-ietf-dprive-phase2-requirements-01
+> 
+> Abstract
+> 
+>    This document provides requirements for adding confidentiality to DNS
+>    exchanges between recursive resolvers and authoritative servers.
+> 
+> 5.  Requirements
+> 
+>    The requirements of different interested stakeholders are outlined
+>    below.
+> 
+> 5.1.  Mandatory Requirements
+> 
+>    1.   Each implementing party should be able to independently take
+>         incremental steps to meet requirements without the need for
+>         close coordination (e.g. loosely coupled)
+
+Yes.
```

Comment:
Do you think we should explain why `yes`, e.g what action can be taken. What are the parties here? recursive and auth? parent zone and zone owner? zone owner and auth name server admin?
Detailing it could possibly help answering future questions from the WG in advance.

Reply:
Yes, makes a lot of sense.

Reply:
I'll try to work that out within this PR, and then we can transplant some variant of that to the draft.

Reply:
pushed, please check